sw/host/opentitanlib/src/app/config/structs.rs

```diff
@@ -118,6 +118,12 @@ pub struct SpiConfiguration {
     pub bits_per_word: Option<u32>,
     /// Data communication rate in bits/second.
     pub bits_per_sec: Option<u32>,
+    /// Which GPIO pin should be used for clock.
+    pub serial_clock: Option<String>,
+    /// Which GPIO pin should be used for signal from debugger to OpenTitan.
```

Comment:
Maybe?
```suggestion
    /// Which GPIO pin should be used for signal from device to host.
```

Reply:
I think that the variable name host_out_device_in already exlaplains that the the signal goes from a SPI "host" to a SPI "device".  I wanted the comment to say they same thing in different terms, by stating that it went from the "debugger" (the hardware device directly controlled by opentitantool) to "OpenTitan" the chip under test.

I think it would add little value to change the description tin "from host to device", (and it definitely should not say "from device to host", as that is the opposite of what it is doing.)